osu.Game/Overlays/Profile/Header/Components/FriendsButton.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Users;
+
+namespace osu.Game.Overlays.Profile.Header.Components
+{
+    public class FriendsButton : ProfileHeaderStatisticsButton
+    {
+        public readonly Bindable<User> User = new Bindable<User>();
+
+        public override string TooltipText => "followers";
```

Comment:
on second thought this is confusing now - this used to say "friends", so there was no confusion, but this is now saying "followers" on a `FriendsButton` - I know the copy differs on whether you're on your own profile or someone else's, but while we don't have friending implemented, maybe best to rename to `FollowersButton`? dunno, it's a weird one. not sure if friending would go into this component or a separate one.

Reply:
Renamed to `FollowersButton` then